scania-digital-design-system/tegel
# fix(header-cell): changed sorting arrow direction in header cell

**Describe pull-request**  
Current direction of sorting arrow does not align with standard direction found in common  web design libraries. Changed direction of arrow to align with standard sorting conventions. Arrow up is used for ascending order and arrow down is used for descending order. Also replaced hover icon when switching sorting key with arrow up to make it clear what direction the table will sort in on first click.

**How to test**  
1. Go to sorting section of table component in storybook
2. Click on header cell and see direction of arrow on ascending and descending sorting events.


**Screenshots**  
Old sorting:
![Sorting_arrow_old](https://github.com/scania-digital-design-system/tegel/assets/90179390/01e88706-ebde-48df-9150-759d95af178a)

New sorting:
![Sorting_arrow_new](https://github.com/scania-digital-design-system/tegel/assets/90179390/e9bc54e3-c904-4523-b445-2b20a7d4560e)

Material design documentation about sorting in tables:
![Screenshot 2023-12-06 at 19 50 50](https://github.com/scania-digital-design-system/tegel/assets/90179390/c2b9b580-065d-4a89-b8d5-be8c1fbc4473)
![Screenshot 2023-12-06 at 19 51 29](https://github.com/scania-digital-design-system/tegel/assets/90179390/2d3019ee-b5e7-41ee-8b94-fb7e2233c5cf)



**Additional context**  
[Material design table documentation](https://m2.material.io/components/data-tables#anatomy)
